### DISCUSSION TITLE
Can you please add dscrete cosine transform and/or wavelet transform (not wavelet decompose because you already have it)

### DISCUSSION THREAD

**Discussion by Wowtones** [photopea/photopea#6471] [Ideas]
Sorry nothing here

**Comment by photopea** [comment]
We already have the Fourier transform. Do you have any example of what you want the Cosine transform to work like, and to look like? Can some software do what you want, and how does it look like?

**Reply by Wowtones** [reply]
No software can and for examples search it up

**Reply by photopea** [reply]
So no software can do it?

**Comment by LeoAndSabrina** [comment]
hi
